vm_server/accept/server.py

```diff
@@ -68,18 +100,23 @@ def move_output(task_request, task_response):
     task_request: an object of TaskResponse() that is sent in the request
     task_response: an object of TaskResponse() that will be sent back
   """
-  logging.debug("Moving the output path", end=" ")
-  logging.debug("to the specified output path")
-  source_path = "..\\execute\\action\\output\\"
-  destination_path = task_request.output_path
+  logging.debug("Moving the output path to the specified output path")
+  current_path = os.getcwd()
+  source_path = Path(current_path + "\\" + EXECUTE_ACTION_DIR + OUTPUT_DIR)
+  if source_path.exists() is False:
+    os.mkdir(source_path)
+  destination_path = Path(current_path + "\\" + task_request.output_path)
+  if destination_path.exists() is False:
+    os.mkdir(destination_path)
   files = os.listdir(source_path)
   try:
     for file in files:
-      shutil.move(source_path + file, destination_path)
+      shutil.move(os.path.join(source_path, file),
+                  os.path.join(destination_path, file))
   except Exception as exception:  # catch errors if any
     logging.exception(str(exception))
-    logging.debug("Error moving the output files", end=" ")
-    logging.debug("to the specified output directory")
+    logging.debug("Error moving the output files \
+                   to the specified output directory")
```

Comment:
this can go in the previous line?

Reply:
This is in the separate line because in one line it exceeds 80 characters and pylint catches this